test/test_suite.py

```diff
@@ -564,6 +567,11 @@ class SequenceTest(_SequenceTest):
     pass
 
 
+@pytest.mark.skip("Spanner doesn't support quotes in table names.")
+class QuotedNameArgumentTest(_QuotedNameArgumentTest):
+    pass
```

Comment:
This test tries to create tables, which name contain quotes. It's not supported and gives a syntax error even if the name was escaped.

https://cloud.google.com/spanner/docs/data-definition-language#naming_conventions